python/ray/serve/deployment_graph.py

```diff
@@ -1,8 +1,13 @@
 import json
-from ray.experimental.dag.class_node import ClassNode
-from ray.experimental.dag.function_node import FunctionNode
-from ray.experimental.dag import DAGNode
+from ray.experimental.dag.class_node import ClassNode  # noqa: F401
+from ray.experimental.dag.function_node import FunctionNode  # noqa: F401
+from ray.experimental.dag import DAGNode  # noqa: F401
 from ray.util.annotations import PublicAPI
+from ray.serve.pipeline.deployment_node import DeploymentNode  # noqa: F401
+from ray.serve.pipeline.deployment_function_node import (  # noqa: F401
+    DeploymentFunctionNode,
+)
```

Comment:
why do we still need these deploymentnode and deploymentfunctionnode types?

Reply:
looks like it's just for typechecks in test. i just removed it.

Reply:
At least within this PR to keep it lightweight -- we need the `DeploymentNode` implementation that can only be cut down more lightweight in later PRs.